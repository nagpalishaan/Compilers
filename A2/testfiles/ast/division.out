Program
  List
    Function
      IdDecl ID="m"
      Opt
      Opt
        List
          AssignVar
            IdDecl ID="a"
            Numeral NUMERAL="2"
          AssignVar
            IdDecl ID="b"
            Numeral NUMERAL="3"
          AssignVar
            IdDecl ID="c"
            Div
              IdUse ID="a"
              IdUse ID="b"
