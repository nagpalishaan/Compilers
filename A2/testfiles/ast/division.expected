Program
  List
    Function
      IdDecl ID="m"
      List
      List
        AssignVar
          IdDecl ID="a"
          Numeral NUMERAL="2"
        AssignVar
          IdDecl ID="b"
          Numeral NUMERAL="3"
        AssignVar
          IdDecl ID="c"
          Div
            IdUse ID="a"
            IdUse ID="b"
